components/atom/button/src/Button.js

```diff
@@ -14,13 +14,12 @@ const Button = ({
 }) => {
   if (isSubmit) attrs.type = 'submit'
   if (isButton) attrs.type = 'button'
+
+  const defaultRel = target === '_blank' ? 'noopener' : undefined
+  const rel = attrs.rel || defaultRel
```

Comment:
wouldn't this achieve the same and avoid an error if `rel` is not present on `attrs`? 

```suggestion
  const {rel = defaultRel} = attrs
```

Reply:
I like the more traditional way

![image](https://user-images.githubusercontent.com/537921/94688972-93eb9180-032e-11eb-967d-928e90c48eda.png)

vs

![image](https://user-images.githubusercontent.com/537921/94689058-ae256f80-032e-11eb-81b1-286c529891a0.png)

Reply:
gotcha... I'm just so used to assigning while destructuring...